packages/amplify-codegen-appsync-model-plugin/src/languages/java-declaration-block.ts

```diff
@@ -0,0 +1,264 @@
+    const args = method.args.map(arg => this.printMember(arg)).join(', ');
+    const comment = method.comment ? transformComment(method.comment) : '';
+    const possibleException = method.exception && method.exception.length ? ` throws ${method.exception.join(', ')}` : '';
+    return [
+      comment,
+      [pieces.join(' '), '(', args, ')', possibleException, ' ', '{'].filter(p => p).join(''),
+      '\n',
+      indentMultiline(method.implementation),
+      '\n',
+      '}',
+    ]
+      .filter(p => p)
+      .join('');
+  }
+
+  addClassMember(
+    name: string,
+    type: string,
+    value: string,
+    typeAnnotations: string[] = [],
+    access: Access = 'protected',
+    flags: MemberFlags = {}
+  ): JavaDeclarationBlock {
+    this._members.push({
+      name,
+      type,
+      value,
+      annotations: typeAnnotations,
+      access,
+      flags: {
+        final: false,
+        transient: false,
+        volatile: false,
+        static: false,
+        ...flags,
+      },
+    });
+
+    return this;
+  }
+
+  addClassMethod(
+    name: string,
+    returnType: string | null,
+    impl: string,
+    args: Partial<ClassMember>[] = [],
+    returnTypeAnnotations: string[] = [],
+    access: Access = 'protected',
+    flags: MemberFlags = {},
+    methodAnnotations: string[] = [],
+    exception?: string[],
+    comment?: string
+  ): JavaDeclarationBlock {
+    this._methods.push({
+      name,
+      returnType,
+      implementation: impl,
+      args,
+      returnTypeAnnotations,
+      access,
+      flags: {
+        final: false,
+        transient: false,
+        volatile: false,
+        static: false,
+        ...flags,
+      },
+      methodAnnotations: methodAnnotations || [],
+      exception,
+      comment,
+    });
+
+    return this;
+  }
+
+  public get string(): string {
+    let result = '';
+
+    if (this._kind) {
+      let name = '';
+
+      if (this._name) {
+        name = this._name;
+      }
+
+      let extendStr = '';
+      let implementsStr = '';
+      let annotatesStr = '';
+      const final = this._final ? ' final' : '';
+      const isStatic = this._static ? ' static' : '';
+
+      if (this._extendStr.length > 0) {
+        extendStr = ` extends ${this._extendStr.join(', ')}`;
+      }
+
+      if (this._implementsStr.length > 0) {
+        implementsStr = ` implements ${this._implementsStr.join(', ')}`;
+      }
+
+      if (this._annotations.length > 0) {
+        annotatesStr = this._annotations.map(a => `@${a}`).join('\n') + '\n';
+      }
+
+      result += `${annotatesStr}${this._access}${isStatic}${final} ${this._kind} ${name}${extendStr}${implementsStr} `;
+    }
+
+    const members = this._members.length
+      ? indentMultiline(stripIndent(this._members.map(member => this.printMember(member) + ';').join('\n')))
+      : null;
+    const methods = this._methods.length
+      ? indentMultiline(stripIndent(this._methods.map(method => this.printMethod(method)).join('\n\n')))
```

Comment:
Are the two new lines intentional?

Reply:
one is `members` and  the other one is `methods`